src/github/middleware.ts

```diff
@@ -208,7 +208,7 @@ export default (
 			const util = getJiraUtil(jiraClient);
 
 			try {
-				await callback(context, jiraClient, util);
+				await callback(context, jiraClient, util, gitHubInstallationId);
```

Comment:
I passed the installation ID into the webhook processing function so that I could add it to the SQS message without doing another database call.